sass/index.scss

```diff
@@ -758,6 +881,72 @@ footer {
         }
     }
 
+    // Prizes
+    #prizes {
+        $tape-offset: 12px;               // how far into the card is the tape placed?
+        $overflow-size: 35px;             // how far does the outer card go into the section margins?
+        $inner-card-overflow-size: 20px;  // how far does the inner card go into the outer card's margins?
+
+        margin-top: -60px;
+
+        #prizes-card {
+            margin-left: -$overflow-size;
+            padding: $overflow-size $overflow-size ($overflow-size + 40px) $overflow-size;
+            width: calc(100% + (2 * #{$overflow-size}));  // #{...} is Sass interpolation
```

Comment:
nit: no need to comment sass interpolation, is fairly common feature of SASS + easy to google if needed